pkg/streamingpromql/functions.go

```diff
@@ -43,17 +43,17 @@ func SingleInputVectorFunctionOperatorFactory(name string, metadataFunc function
 	}
 }
 
-// TransformationFunctionOperatorFactory creates an InstantVectorFunctionOperatorFactory for functions
+// SingleInputVectorTransformationFunctionOperatorFactory creates an InstantVectorFunctionOperatorFactory for functions
 // that have exactly 1 argument (v instant-vector), and drop the series __name__ label.
 //
 // Parameters:
 //   - name: The name of the function.
 //   - seriesDataFunc: The function to handle series data
-func TransformationFunctionOperatorFactory(name string, seriesDataFunc functions.InstantVectorFunction) InstantVectorFunctionOperatorFactory {
+func SingleInputVectorTransformationFunctionOperatorFactory(name string, seriesDataFunc functions.InstantVectorFunction) InstantVectorFunctionOperatorFactory {
 	return SingleInputVectorFunctionOperatorFactory(name, functions.DropSeriesName, seriesDataFunc)
 }
 
-// LabelManipulationFunctionOperatorFactory creates an InstantVectorFunctionOperator for functions
+// SingleInputVectorLabelManipulationFunctionOperatorFactory creates an InstantVectorFunctionOperator for functions
```

Comment:
Same as above - I think we can drop the `SingleInput` and "just" (😅) call it `InstantVectorLabelManipulationFunctionOperatorFactory`